demo/index.html

```diff
@@ -12,11 +12,12 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
-  <script src="../webcomponentsjs/webcomponents.js"></script>
-  <link href="paper-shadow.html" rel="import">
+  <script src="../../webcomponentsjs/webcomponents.js"></script>
+  <link href="../../layout/layout.html" rel="import">
+  <link href="../paper-shadow.html" rel="import">
 
   <style>
-      body {
+    body {
       font-family: RobotoDraft, 'Helvetica Neue', Helvetica, Arial;
```

Comment:
I know this was here before you, but where does `RobotoDraft` come from? It doesn't seem to be included anywhere.